test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -2674,27 +2675,40 @@ class XdsEnd2endTest : public ::testing::TestWithParam<TestType> {
     // Variables for synchronization
     absl::Mutex mu;
     absl::CondVar cv;
+    size_t started = 0;
     size_t completed = 0;
     // Set-off callback RPCs
     for (size_t i = 0; i < num_rpcs; i++) {
+      {
+        // xDS LB policy `xds_cluster_impl_experimental` has a default
```

Comment:
I don't think this makes sense.  The whole point of `SendConcurrentRpcs()` is to start a specific number of concurrent RPCs, so changing it to start less than that number is basically breaking it.

If we have tests where we're hitting the max_concurrent_requests limit, then let's instead change those tests to raise that limit in the xDS config.

Reply:
The suggestion is implemented in https://github.com/grpc/grpc/pull/26626.

My original goal of `SendConcurrentRpcs` is sending a certain number of RPCs in a concurrent fashion, so the test cases can run faster. What do you think about adding a `SendConcurrentRpcsWithThrottle` method?

Reply:
I wouldn't be opposed to it if we had a need for it.  But I don't really see the need.

I think #26626 looks good.